java/openjdk10/Portfile

```diff
@@ -4,29 +4,48 @@ PortSystem       1.0
 
 name             openjdk10
 version          10.0.2
+set              openjdk_version 10
+
+subport openjdk11 {
+    version      11
+    set          openjdk_version 11
+    set          build 28
+}
 
 categories       java devel
 maintainers      {breun.nl:nils @breun} openmaintainer
 platforms        darwin
 license          GPL-2
 supported_archs  x86_64
 
-description      Open Java Development Kit 10
+description      Open Java Development Kit ${openjdk_version}
 
 long_description Production-ready, free and open-source build of the Java \
                  Development Kit, an implementation of the Java Standard \
-                 Edition (SE) 10 Platform. OpenJDK is the official reference \
+                 Edition (SE) ${openjdk_version} Platform. OpenJDK is the official reference \
```

Comment:
I'd use `${version}` instead of `${openjdk_version}` here, as that's also what https://jdk.java.net/10/ says.